bundles/org.eclipse.osgi/META-INF/MANIFEST.MF

```diff
@@ -7,10 +7,12 @@ Export-Package: org.eclipse.core.runtime.adaptor;x-friends:="org.eclipse.core.ru
   uses:="org.eclipse.osgi.report.resolution,
    org.osgi.framework.wiring,
    org.eclipse.osgi.framework.eventmgr,
+   org.apache.felix.resolver,
    org.osgi.framework.startlevel,
    org.osgi.framework,
    org.osgi.framework.hooks.resolver,
    org.osgi.service.resolver,
+   org.eclipse.osgi.internal.container,
```

Comment:
@laeubi this one also is wrong or it is an indication that we are leaking internal types into the API of `org.eclipse.osgi.container`

Reply:
Lets see if it is fixed by https://github.com/eclipse-pde/eclipse.pde/pull/1830 otherwise I have to take a deeper look.